exchanges/binance.js

```diff
@@ -164,25 +164,25 @@ Trader.prototype.getFee = function(callback) {
 };
 
 Trader.prototype.getTicker = function(callback) {
-  var setTicker = function(err, data) {
-    log.debug(`[binance.js] entering "getTicker" callback after api call, err: ${err} data: ${JSON.stringify(data)}`);
-    if (err) return callback(err);
+    var setTicker = function(err, data) {
```

Comment:
could you stick to 2 spaces indenting? Thanks!

Reply:
My editor is set to 4 space and sometimes it overrides the existing indenting, will fix.